lib/Operation.php

```diff
@@ -106,11 +106,11 @@ public function isTaggingPath(IStorage $storage, string $file): bool {
 			list($folder) = explode('/', $file, 2);
 			return $folder === 'files';
 		} else {
-			list($folder) = explode('/', $file, 2);
+			[$folder, $subPath] = explode('/', $file, 2);
```

Comment:
There might not be a slash, so this needs more fixing

Reply:
Pushed a commit to return early so we don't try to explode the file path if if just contains the base `__groupfolders` path